front-page.php

```diff
@@ -0,0 +1,348 @@
+        </div>
+
+      </a>
+
+    </article>
+
+<?php 
+    }
+?>
+
+  </section>
+
+<?php 
+  } 
+  wp_reset_postdata(); 
+?>
+
+
+  <section class="row"> <!-- Puta portadazza, Noticias, Ads -->
+
+    <div class="col s8">
+
+<?php 
+  // WP_Query arguments
+  $args = array (
+    'category_name'          => 'puta-portadazza',
+    'posts_per_page'         => '1',
+  );
+
+  // The Query
+  $puta_query = new WP_Query( $args );
+
+  $category = get_category_by_slug( 'puta-portadazza' );
+  $cat_name = $category->cat_name;
+  $cat_id = $category->term_id;
+  $cat_link = get_category_link( $cat_id );
+
+  // The Loop
+  if ( $puta_query->have_posts() ) { 
+?>
+
+      <div class="feed-category">
+        <a href="<?php echo esc_url( $cat_link ); ?>">
+          <span class="rotate-text">
+            <?php echo $cat_name; ?>
+          </span>
+        </a>
+      </div>
+
+<?php
+    while ( $puta_query->have_posts() ) {
+      $puta_query->the_post();
+
+      $puta_id = $post->ID;
+?>
+      <article <?php post_class(); ?> id="post-<?php the_ID(); ?>">
+
+        <a href="<?php the_permalink() ?>">
+
+          <h3 class="featured-title">
+            <?php the_title(); ?>
+          </h3>
+
+          <?php the_post_thumbnail(); ?>
+
+        </a>
+
+        <?php the_excerpt(); ?>
+
+      </article>
+<?php 
+    }
+  } 
+  wp_reset_postdata(); 
+?>
+    </div>
+
+    <div class="col s8">
+
+<?php 
+  // WP_Query arguments
+  $args = array (
+    'category_name'          => 'noticias',
+    'posts_per_page'         => '5',
+  );
+
+  // The Query
+  $noticias_query = new WP_Query( $args );
+
+  $category = get_category_by_slug( 'noticias' );
+
+  // The Loop
+  if ( $noticias_query->have_posts() && $category ) {
+
+    $cat_name = $category->cat_name;
+    $cat_id = $category->term_id;
+    $cat_link = get_category_link( $cat_id );
+
+  ?> 
+
+      <div class="feed-category">
+        <a href="<?php echo esc_url( $cat_link ); ?>">
+          <span class="rotate-text">
+            <?php echo $cat_name; ?>
+          </span>
+        </a>
+      </div>
+       
+<?php
+    while ( $noticias_query->have_posts() ) {
+      $noticias_query->the_post();
+?>
+      <article <?php post_class('noticias'); ?> id="post-<?php the_ID(); ?>">
+
+        <a href="<?php the_permalink() ?>">
+
+          <h3 class="noticias-title">
+            <?php the_title(); ?>
+          </h3>
+
+          <?php the_post_thumbnail(); ?>
+
+        </a>
+
+      </article>
+<?php 
+    }
+  } 
+  wp_reset_postdata(); 
+?>
+
+    </div>
+
+    <div class="col s8">
+
+      <img src="https://placeholdit.imgix.net/~text?txtsize=50&txt=AD&w=400&h=400">
+
+      <img src="https://placeholdit.imgix.net/~text?txtsize=50&txt=AD&w=400&h=400">
+      
+    </div>
+
+  </section>
+
+
+  <?php get_template_part('partials/twitter'); ?><!-- Twitter feed -->
+
+
+  <?php 
+    // WP_Query arguments
+    $exclude_array = array( $featured_id, $puta_id );
```

Comment:
really depends on wtf noticias are. Are they latest posts pulled up top so quicker to be seen and then duplicated in the main grid? Or are they a specific less important cat that doesn't feature in the feed on the home [which would cause all sorts of issues when we go to /page/2]